packages/catalog-process/src/services/catalogService.ts

```diff
@@ -491,17 +501,19 @@ export function createEserviceLogic({
   return toCreateEventEServiceAdded(newEService);
 }
 
-export function updateEserviceLogic({
+export async function updateEserviceLogic({
```

Comment:
This function should not be marked as async

Reply:
Fixed, thanks! Sorry about that: with a previous edit I made a change that required turning this into an `async` function returning a `Promise`; then I reverted that change, but I forgot to revert this as well.